hotline/species/models.py

```diff
@@ -73,3 +73,18 @@ class Meta:
 
     def __str__(self):
         return "%s%s" % (self.name, " (%s)" % self.scientific_name if self.scientific_name else "")
+
+    def to_json(self):
```

Comment:
whoops. Nope. This should have been deleted with the json calls in the view. Thanks for catching it.